### DISCUSSION TITLE
Ponto de avaliação de onde o `status.value` deve ser comparado

### DISCUSSION THREAD

**Discussion by Rafael-Conde** [roboticatera/locomotiva#11] [Ideas]
Eu tava pensando, e pode ser que não tenha problema de envolvermos o resto do código em um if grande, o pronto que o @enzolfernandes colocou é importante e talvez seja interessante evitarmos colocar `delay`s dentro do código, para que ele não demore pra parar. Uma coisa que eu fico pensando, é que colocar ele [`status.value == start`](https://github.com/roboticatera/locomotiva/blob/0595c7ab792151a36857fcfd4849f1e03eef9573/tests/geral/geral.ino#L155) dentro das funções de comando é mudar um código em função de algo externo a ele, eu acho que isso pode ser mais um sinal de que podemos melhorar a interface do código. a curto prazo, eu acho essa solução válida, mas talvez isso pode ser algo pensado com mais calma no futuro.

Eu me refiro como por exemplo na [linha 155 do teste geral](https://github.com/roboticatera/locomotiva/blob/0595c7ab792151a36857fcfd4849f1e03eef9573/tests/geral/geral.ino#L155)

**Comment by enzolfernandes** [comment]
Eu também estava afim de tirar todos os delays, mas não sei como manter a informação visual para o estado do microstart.
